9.4/docker-entrypoint.sh

```diff
@@ -9,9 +15,17 @@ if [ "$1" = 'postgres' ]; then
 	
 	if [ -z "$(ls -A "$PGDATA")" ]; then
 		gosu postgres initdb
-		
-		sed -ri "s/^#(listen_addresses\s*=\s*)\S+/\1'*'/" "$PGDATA"/postgresql.conf
-		
+
+        alter_system "listen_addresses" "*"
+
+        conf_prefix="POSTGRES_CONF_"
+        for env_name in $(env | cut -d= -f1 | grep "$conf_prefix.*"); do
+            value=$env_name
```

Comment:
Does this actually work? I'd think you would need to do `eval value="\$${env_name}"` or something.